src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -972,10 +980,24 @@ export function getVirtualStudyDescription(
                     genomicDataFilter.profileType
                 );
                 const name = attributeNamesSet[uniqueKey];
+                const dataFilterValues = genomicDataFilter.values;
+                const dataType = getDataTypeByDataFilterValues(
+                    dataFilterValues
+                );
+
                 if (name) {
                     filterLines.push(
                         `- ${name}: ${intervalFiltersDisplayValue(
-                            genomicDataFilter.values,
+                            dataType === DataType.NUMBER
+                                ? dataFilterValues
+                                : _.map(dataFilterValues, d =>
+                                      _.assign(d, {
```

Comment:
assign mutates in place, updating the target (d).  is that really what we want to do?  should we be producing something new?